tabbycat/standings/tests/test_standings.py

```diff
@@ -45,7 +44,7 @@ def tearDown(self):
 
     def get_standings(self, generator):
         with suppress_logs('standings.metrics', logging.INFO):
-            standings = generator.generate(self.tournament.team_set.all())
+            standings = generator.generate(self.tournament.team_set.all(), round=Round.objects.get(seq=2))
```

Comment:
Oh, interesting. But I think this would make all tests with the ignorable debate vacuously pass—in particular, `TestStandingsWithEliminationRound` (the other subclass of `IgnorableDebateMixin`) would pass solely because of this `round=` argument. Even if the elimination round filter failed, the `round=` filter would make it look like it had worked.

Similarly, all the tests in `TestStandingsWithUnconfirmedBallotSubmission` are specifically intended to test that unconfirmed ballots are excluded _because they're unconfirmed_, and this would make all other tests inherited from `TestTrivialStandings` vacuously pass too.

I had to think for a while (again!) about whether #1446 is expected behaviour, and I went back and forth three times 😂 but I think you're right? The unconfirmed ballot should not count in draw strength, but the confirmed results of their opponents from all rounds should count, is that what you're saying? And callers should pass in the `round=` argument if they want unfinished rounds to be excluded, as `DrawManager` already does?

If that's the case then I would instead override `test_draw_strength()` and `test_draw_strength_speaks()` with the new expected results, and add a comment to explain why this is the expected result. So (without having actually tried it) for `test_draw_strength()` it would be `[0, 6]`, and for `test_draw_strength_speaks()` it would be `[591, 609]`, I think? Thoughts?